resources/cmake/ConnextDdsAddSecurityArtifacts.cmake

```diff
@@ -0,0 +1,186 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+
+#[[.rst:
+.. _connextdds_add_security_artifacts:
+
+ConnextDdsAddSecurityArtifacts
+------------------------------
+
+Generate the artifacts needed for secure examples.
+
+``connextdds_add_security_artifacts()``
+
+It is based on:
+ - The OpenSSL configuration files in the resources/security/<ca> directory.
+
+ - The Governance and Permissions files in the resources/xml directory.
+
+So far this means:
+
+ - One ECDSA CA and two peers.
+
+ - RTPS and metadata protection kind set to ENCRYPT.
+
+ - Publishing and subscribing to any domain and topic is allowed, only for those
+   peers.
+
+#]]
+
+include(UseOpenSSL)
+
+
+function(connextdds_add_security_artifacts)
+    # Directory where the security artifacts are.
```

Comment:
I'm not sure about this. Is where they are or where they will be generated?

Reply:
This is where the source files are. We use them to generate the security artifacts (signed XML and certificates).
I replaced the comment with:
```
    # Directory where the source files used to generate the security artifacts
    # are. These are .cnf openssl configuration files (to generate the
    # certificates) and the XML Governance and Permission Documents (before
    # signing them).
```